app/main/helpers/validation.py

```diff
@@ -62,9 +62,6 @@ def get_error_message(self, question_id, message_key):
         return default_messages.get(
             message_key, 'There was a problem with the answer to this question')
 
```

Comment:
Sorry, it wasn't clear from the diff that it was defined further on in the base class.